app/scripts/modules/core/application/config/serializeApplicationSection.directive.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.application.config.serialize.directive', [
+    require('angular-ui-router'),
+    require('./../../serialize/serialize.write.service.js'),
+    require('../../confirmationModal/confirmationModal.service.js'),
+    require('../../overrideRegistry/override.registry.js'),
+  ])
+  .directive('serializeApplicationSection', function (overrideRegistry) {
```

Comment:
I think we're using components instead of directives these days. Please check with @danielpeach to get some direction on the right way to go here.